web-api/terraform/modules/ci-cd/ci-cd-permissions.tf

```diff
@@ -85,6 +85,22 @@ resource "aws_iam_policy" "ci_cd_policy" {
       ],
       "Resource": "*"
     },
+    {
+      "Sid": "CognitoIdentityPool",
+      "Effect": "Allow",
+      "Action": [
+        "cognito-identity:*"
+      ],
+      "Resource": "arn:aws:cognito-identity:us-east-1:${data.aws_caller_identity.current.account_id}:identitypool/*"
+    },
+    {
+      "Sid": "CloudWatchRUM",
+      "Effect": "Allow",
+      "Action": [
+        "rum:*"
```

Comment:
We had to keep these wildcard Actions because we were receiving errors stating that the overall policy size is reaching AWS size limits.